addons/base_import/static/src/xml/base_import.xml

```diff
@@ -315,4 +314,33 @@ lines composed only of empty cells">
             </div>
         </div>
     </t>
+
+    <div t-name="base_import.progressDialog" class="o_import_progress_dialog text-white">
+        <span class="fa fa-spin fa-circle-o-notch fa-2x mb-2"/>
+        <div t-if="isBatch">
+            <div class="o_import_progress_dialog_batch">
+                <span t-esc="task"/> batch <span class="o_import_progress_dialog_batch_count">1</span> out of <span t-esc="totalBatch"/>...
```

Comment:
Just to be complete, we could have a counterBatch instead of hardcoding "1" . I know it is then updated through JS, but hey we could pretend doing automatically updated DOM based phantom jquery mixin abstract stuff.